src/chi/javascript/components/mobile-navigation.js

```diff
@@ -441,18 +445,21 @@ class MobileNavigation extends Component {
   }
 
   dispose() {
+    this._firstLevelDrawer.dispose();
     this._config = null;
     this._elem = null;
-    this._firstLevelDrawer = null;
```

Comment:
Please keep this line. First call dispose, as you're currently doing, and then, set it to null or delete it. E.g.
```Javascript
delete this._firstLevelDrawer;
```

Reply:
Done, thank you!